server/app.js

```diff
@@ -22,9 +25,9 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, '../public/dist')));
 
 app.use('/users', users);
-app.use('/sessions', sessions);
+app.use('/authenticate', authenticate);
 
-app.use('*', (req, res, next) => {
+app.use('/', (req, res, next) => {
```

Comment:
We might have to change this back to a wildcard (*), I'll try it after this merges and see what happens.